### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by breisfeld** [cantino/mcfly#75]
Platform: Debian 10.2 on wsl on Windows10 

```
$ rustc --version                                                                                                                   127 ↵
rustc 1.39.0 (4560ea788 2019-11-04)
```

```
$ cargo install --path .  

...

error[E0004]: non-exhaustive patterns: `BackTab` not covered
   --> src/interface.rs:313:23
    |
313 |                 match c.unwrap() {
    |                       ^^^^^^^^^^ pattern `BackTab` not covered
    |
    = help: ensure that all possible cases are being handled, possibly by adding wildcards or more match arms
```

**Comment by cantino** [comment]
Thank you. I didn't realize other Windows had other keys that I wasn't accounting for. Would you mind seeing if #73, which I just merged, fixes it for you?

**Comment by breisfeld** [comment]
It now builds fine. Thank you.

**Comment by cantino** [comment]
Awesome, fixed in v0.3.6 then.
